src/settings.c

```diff
@@ -339,7 +391,10 @@ static void settings_msg_callback(u16 sender_id, u8 len, u8 msg[], void* context
 
 static void settings_read_by_index_callback(u16 sender_id, u8 len, u8 msg[], void* context)
 {
-  (void)sender_id; (void) context;
+  (void) context;
+
+  if (sender_id != 0x42)
```

Comment:
Lets make sure to update the LIBSBP document to explain what a sender ID of 0x42 will allow on the device.  I sort of think the read functions should be allowed from any sender id, while the write should be disallowed.